module/epochs/epoch_lookup_test.go

```diff
@@ -170,44 +160,49 @@ func (suite *EpochLookupSuite) TestEpochForViewWithFallback_CurrNextPrev() {
 // EpochLookup with an initial state of epoch fallback triggered.
 func (suite *EpochLookupSuite) TestEpochForViewWithFallback_EpochFallbackTriggered() {
 	epochs := []epochRange{suite.prevEpoch, suite.currEpoch, suite.nextEpoch}
-	suite.WithLock(func() {
-		suite.epochFallbackTriggered = true
-	})
 	suite.CommitEpochs(epochs...)
 	suite.CreateAndStartEpochLookup()
 	testEpochForViewWithFallback(suite.T(), suite.lookup, suite.state, epochs...)
 }
 
-// TestProtocolEvents_EpochFallbackTriggered tests constructing and subsequently querying
-// EpochLookup, where there is no epoch fallback at construction time,
-// but an epoch fallback happens later via an epoch event.
-func (suite *EpochLookupSuite) TestProtocolEvents_EpochFallbackTriggered() {
-	// initially, only current epoch is committed
-	suite.CommitEpochs(suite.currEpoch)
+// TestProtocolEvents_EpochExtended tests constructing and subsequently querying
+// EpochLookup, where we process an EpochExtended event and expect the latest
+// epoch final view to be updated with the updated final view of the current epoch
+// in the protocol state.
+func (suite *EpochLookupSuite) TestProtocolEvents_EpochExtended() {
+	// previous and current epochs will be committed
+	suite.CommitEpochs(suite.prevEpoch)
+	epoch, extension := suite.epochFixtureWithExtension(suite.currEpoch.counter, suite.currEpoch.firstView, suite.currEpoch.finalView+1000)
+	suite.epochQuery.Add(epoch)
```

Comment:
If we operate directly on the `EpochExtension`, I think we can simplify the mocking to:
```suggestion
	suite.CommitEpochs(suite.prevEpoch, suite.currEpoch)
```